Deprecate metrics (#4739)
huggingface/datasets: docs/source/about_metrics.mdx

# All about metrics

<Tip warning={true}>

> Metrics will soon be deprecated in 🤗 Datasets. To learn more about how to use metrics, take a look at our newest library 🤗 [Evaluate](https://huggingface.co/docs/evaluate/index)! In addition to metrics, we've also added more tools for evaluating models and datasets.

Reviewer: ```suggestion
Metrics are deprecated in 🤗 Datasets. To learn more about how to use metrics, take a look at our newest library 🤗 [Evaluate](https://huggingface.co/docs/evaluate/index)! In addition to metrics, we've also added more tools for evaluating models and datasets.
```

Author: Done.
